src/core/core.controller.js

```diff
@@ -246,15 +247,16 @@ class Chart {
 		// Before init plugin notification
 		plugins.notify(me, 'beforeInit');
 
-		helpers.dom.retinaScale(me, me.options.devicePixelRatio);
-
-		me.bindEvents();
+		// helpers.dom.retinaScale(me, me.options.devicePixelRatio);
```

Comment:
Why is this commented out?

Reply:
Leftovers, thats only needed in the `else` lower.